Mathlib/Order/Monotone/Basic.lean

```diff
@@ -1256,3 +1256,10 @@ theorem const_strictMono [Nonempty β] : StrictMono (const β : α → β → α
 #align function.const_strict_mono Function.const_strictMono
 
 end Function
+
+theorem prod_map_iterate {X Y : Type _} (S : X → X) (T : Y → Y) (n : ℕ) :
+    (Prod.map S T)^[n] = Prod.map S^[n] T^[n] := by
+  induction' n with n hn
```

Comment:
What about switching from `induction'` to `induction`?

Reply:
Done. Thanks.